scram-tools.file/tools/xrdcl-record/xrdcl-record.xml

```diff
@@ -0,0 +1,7 @@
+<tool name="xrdcl-record" version="@TOOL_VERSION@">
+  <lib name="XrdClRecorder-5"/>
+  <client>
+    <environment name="XRDCL-RECORD_BASE" default="@TOOL_ROOT@"/>
+    <environment name="LIBDIR"            default="$xrdcl-record_BASE/lib64"/>
+  </client>
```

Comment:
As this just provides a plugin for xrootd, so I think we do not need to provide LIBDIR abd also remove `<lib name="..."/>` . Instead just add `<runtime name="XRDCL_RECORDER_PLUGIN"       value="${XRDCL-RECORD_BASE}/lib64/libXrdClRecorder-5.so"/>` . this way one someone wants to create the [recorder.conf](https://github.com/xrootd/xrdcl-record) then they can use `$XRDCL_RECORDER_PLUGIN`